orc8r/cloud/go/obsidian/swagger/poll.go

```diff
@@ -36,9 +36,9 @@ func GetCombinedSpec(yamlCommon string) (string, error) {
 
 	var yamlSpecs []string
 	for _, s := range servicers {
-		yamlSpec, err := s.GetSpec()
+		yamlSpec, err := s.GetPartialSpec()
 		if err != nil {
-			// Swallow GetSpec error because the polling should continue
+			// Swallow GetPartialSpec error because the polling should continue
```

Comment:
Nit: you can just say "Swallow error...", no need to include the name of the function